packages/module/patternfly-docs/content/examples/CatalogView.md

```diff
@@ -1,8 +1,10 @@
 ---
-id: Catalog view demo
+id: Overview
+title: Catalog view overview
 section: extensions
 subsection: Catalog view
 source: react
```

Comment:
Wondering if this is why the docs page looks different from the ChatBot one - do we need to update the source so you don't get "Untitled example" here?:

<img width="910" height="195" alt="Screenshot 2025-11-24 at 11 09 17 AM" src="https://github.com/user-attachments/assets/0f50eafe-4523-4374-af72-679bc82601fd" />

ChatBot's source is source: ChatBot fwiw, but I am far from an expert on the docs framework.